cl/api/templates/rest-docs-vlatest.html

```diff
@@ -986,6 +986,23 @@ <h3 id="alerts-endpoint">{% url "alert-list" version="v3" %}</h3>
   "{% get_full_host %}{% url "alert-detail" version="v3" pk=9999 %}"</pre>
     <p>As always, for more details place an <code>OPTIONS</code> request to the endpoint.</p>
 
+    <h3 id="docket-alerts-endpoint">{% url "docket-alert-list" version="v3" %}</h3>
+    <p><a href="{% url "alert_help" %}#recap-alerts">Docket Alerts</a> can be made, viewed, modified, and deleted using this endpoint. This can be useful for organizations using <a href="{% url 'webhooks_docs' %}#docket-alerts">Docket Alert Webhooks</a> or if you need docket alerts on many dockets. To create a docket alert, send:
+    </p>
+    <ul>
+      <li><strong><code>docket</code></strong> &mdash; The docket to subscribe to.</li>
+      <li><strong><code>alert_type</code></strong> &mdash; Optional, by default <code>1</code> Subscription or <code>0</code> Unsubscription (docket alert disabled).</li>
+    </ul>
+    <p>For more details place an <code>OPTIONS</code> request to the endpoint.</p>
```

Comment:
We need to explain what an unsubscription is in this context. How's this:

```suggestion
    <p><a href="{% url "alert_help" %}#recap-alerts">Docket Alerts</a> can be made, viewed, modified, and deleted using this endpoint. This can be useful for organizations using <a href="{% url 'webhooks_docs' %}#docket-alerts">Docket Alert Webhooks</a> or if you need docket alerts on many dockets.
    </p>
    <p>To create a docket alert, send a POST request with the <code>docket</code> ID you wish to subscribe to.
    </p>
    <p>An optional secondary argument, <code>alert_type</code>, may also be sent. This argument works with our auto-subscribe feature. When auto-subscribe is enabled in your account and your <a href="XXX">@recap.email</a> email address receives a notification from PACER, we will automatically subscribe you to that case. If instead you wish to <em>not</em> get notifications for that case, while keeping auto-subscribe on, you must "unsubscribe" from the case. To do this, create a docket alert with the <code>alert_type</code> set to <code>1</code>, which represents an unsubscription.
    </p>
    <p>For more details place an <code>OPTIONS</code> request to the endpoint.</p>
```

Reply:
Thanks, just tweaked the alert_type for unsubscription that's `0`.